contrib/guix/README.md

```diff
@@ -114,6 +114,12 @@ find output/ -type f -print0 | sort -z | xargs -r0 sha256sum
   depends tree. Setting this to the same directory across multiple builds of the
   depends tree can eliminate unnecessary redownloading of package sources.
 
+* _**BASE_CACHE**_
+
+  Set the depends tree cache for built packages. This is passed through to the
+  depends tree. Setting this to the same directory across multiple builds of the
+  depends tree can eliminate unnecessary building of packages.
```

Comment:
Should this mention that the folder should be unique per major version of Bitcoin Core? See #17248

Reply:
I guess long term the goal is to move the depends cache to the guix gnu store anyway?

Reply:
See: https://github.com/bitcoin/bitcoin/issues/17248#issuecomment-780006662